.github/workflows/deploy_test_web.yaml

```diff
@@ -0,0 +1,84 @@
+name: Deploy Web (Test)
+
+on:
+  push:
+    branches:
+      - build/test
+      - main
+env:
+  NODE_VERSION: "18.16.0"
+  PNPM_VERSION: "8.5.0"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      SSH_PRIVATE_KEY: ${{ secrets.WEB_TEST_SSH_PRIVATE_KEY }}
+      REMOTE_HOST: ${{ secrets.WEB_TEST_REMOTE_HOST }}
+      REMOTE_USER: ${{ secrets.WEB_TEST_REMOTE_USER }}
+      SSL_CERTIFICATE: ${{ secrets.WEB_TEST_SSL_CERTIFICATE }}
+      SSL_CERTIFICATE_KEY: ${{ secrets.WEB_TEST_SSL_CERTIFICATE_KEY }}
+      ENV_FILE: env.test
+    steps:
+      - uses: actions/checkout@v4
+      - name: Maximize build space (ubuntu only)
```

Comment:
It seems redundant. The CI job runs on ubuntu-latest ('ubuntu-22.04'), but the condition uses 'ubuntu-20.04'.